test/spec/runners/operations/runner.ex

```diff
@@ -30,17 +30,29 @@ defmodule OperationsTestRunner do
   def run_test_case(%SpecTestCase{} = testcase) do
     case_dir = SpecTestCase.dir(testcase)
     handler = testcase.handler
+    config = SpecTestUtils.get_config(testcase.config)
 
-    {:ok, pre} = OperationsTestUtils.prepare_test(case_dir, handler, "pre")
+    {:ok, pre} =
+      SpecTestUtils.read_ssz_from_file(
```

Comment:
This one should be `read_ssz_from_file!` since the _pre_ file should always exist, right?